testing/mongo-c-driver/APKBUILD

```diff
@@ -0,0 +1,54 @@
+# Maintainer: Leonardo Arena <rnalrd@alpinelinux.org>
+pkgname=mongo-c-driver
+pkgver=1.14.0
+pkgrel=0
+pkgdesc="Client library written in C for MongoDB"
+url="https://github.com/mongodb/mongo-c-driver"
+arch="all"
+license="Apache-2.0"
+makedepends="openssl-dev cyrus-sasl-dev icu-dev py3-sphinx cmake"
+subpackages="${pkgname}-static ${pkgname}-dev ${pkgname}-doc libbson"
```

Comment:
```suggestion
subpackages="${pkgname}-libs-static ${pkgname}-libs ${pkgname}-dev ${pkgname}-doc"
```

Reply:
Done.
Now `mongo-c-driver-1.14.0-r0.apk` contains empty `/usr/lib` folder.
How it can be removed after package splitting?

Reply:
```
libs() {
	default_libs

	# remove if empty, ignore error (not empty)
	rmdir "${pkgdir}/usr/lib" "${pkgdir}/usr" 2>/dev/null || :
}
```

Reply:
Done.
I made without failback if `/usr/lib` is empty. If package updated and `/usr/lib` folder not empty, then just remove `libs` function.
This allow maintain build script clean.

Also removed `libbson` function
```diff
diff --git a/testing/mongo-c-driver/APKBUILD b/testing/mongo-c-driver/APKBUILD
index 489128af57..04a20e707e 100644  
--- a/testing/mongo-c-driver/APKBUILD
+++ b/testing/mongo-c-driver/APKBUILD
@@ -47,10 +47,4 @@ package() {                                       
        mv "${pkgdir}/usr/share/${pkgname}" "${pkgdir}/usr/share/doc"
 }
            
-libbson() {                                                     
-       pkgdesc="Building, parsing, and iterating BSON documents"
-       mkdir -p "${subpkgdir}/usr/lib"                                                    
-       find "${pkgdir}/usr/lib/" -name "libbson-*" -exec mv "{}" "${subpkgdir}/usr/lib" \;
-}
-                                                                                                                                                                           
 sha512sums="bf2bb835543dd2a445aac6cafa7bbbf90921ec41014534779924a5eb7cbd9fd532acd8146ce81dfcf1bcac33a78d8fce22b962ed7f776449e4357eccab8d6110  mongo-c-driver-1.14.0.tar.gz"
```